packages/grafana-data/src/types/templateVars.ts

```diff
@@ -130,7 +130,7 @@ export interface VariableWithMultiSupport extends VariableWithOptions {
 export interface VariableWithOptions extends BaseVariableModel {
   current: VariableOption | Record<string, never>;
   options: VariableOption[];
-  query: string;
```

Comment:
If this is mandatory, where is it assigned as null? Maybe we can act there instead of allowing this to happen.

I know this is already a try from a more surfaced issue, but just trying to make sure we act on the root cause. If the query must be a defined string, maybe we don't need to make it optional. Sorry to bother :(

Reply:
as far as I understand they used [grabana](https://github.com/K-Phoen/grabana) to generate dashboards and its assigned there (its a 3rd party tool and not sure how much we support/encourage it). Old arch handles it, but scenes break

Reply:
Yes, I'm not saying the JSON doesn't contain it. We can't manage that. But for example, we could do it when we load the JSON. 

When the JSON is loaded, first it goes through DashboardModel, and probably is there when the variable value is not properly initialized if the value coming is null. If we fallback there to an empty string, we might be safe. WDYT?